docs/use_cases/legends-holders.html

```diff
@@ -0,0 +1,323 @@
+        // Manage window size changed
+        themeManager.manageChartResize(myChart, 'barChartSH_chart');
+        // Register the externalization of the tooltip/popup
+        themeManager.externalizePopover();
+        // Display the chart using the configured theme and data.
+        myChart.setOption(themeManager.getChartOptions());
+      </script>
+
+      <div class="card w-100 mt-3">
+        <div class="card-body">
+          <h5 class="card-title">Legends holders for stacked bars example</h5>
+          <p class="card-text">In the following example, we want to display the legends for a first set of stacked bars separately from those for a second set of stacked bars and the legend for a curve.</p>
+          <p class="card-text">To do this, we pass as a parameter to the externalizeLegends() method not a css selector of the legend container but an array of containers defined by the <code>{legendHolderSelector: string; orientation?: 'vertical' | 'horizontal'; serieRef?: string[]}</code> object.</p>
+          <ul>
+            <li><code>legendHolderSelector</code> is the css selector of the legend holder.</li>
+            <li><code>orientation</code> optionally indicates whether captions are displayed horizontally or vertically. By default, the value of the <code>legend.orient</code> option in Apache Echarts will be used if it exists, otherwise it will be <code>'horizontal'</code>.</li>
+            <li>
+              <code>serieRef</code> is an array referencing the series whose legends are to be displayed in this legend container. The reference can be the series name, the series label or simply the stack name in the case of stacked bars.<br />
+              If <code>serieRef</code> is absent, this container will be the default container for non-referenced legends.
+            </li>
+          </ul>
+          <p class="card-text">
+            We'll then have the following code to externalise our three types of legend into three different containers:
+            <code>
+              <pre>
+                themeManager.externalizeLegends(
+                  myChart, 
+                  [{ legendHolderSelector: '#results_stack_legend', serieRef: ['result'], orientation: 'vertical' }, 
+                   { legendHolderSelector: '#goals_stack_legend', serieRef: ['goals'], orientation: 'vertical' }, 
+                   { legendHolderSelector: '#line_legend' }]); 
+              </pre>
+            </code>
+          </p>
+          <div id="stacked_htmlId">
+            <div class="border border-light position-relative">
+              <div class="chart_title">
+                <h4 class="display-4 mx-3 mb-1 mt-3">Title</h4>
+                <h5 class="display-5 mx-3 mb-1 mt-0">Sub-Title</h5>
+              </div>
+              <div id="barLine_holder">
+                <div id="barLine_chart" style="width: 100%; height: 50vh" class="position-relative"></div>
+              </div>
+
+              <div class="row mx-3">
```

Comment:
```suggestion
              <div class="row mx-2">
```
Just to align with titles